### PR TITLE
Update altspace.json

### PR DESCRIPTION
**Pull request by GrumpyCrouton**

### CODE REVIEW

File: decks/altspace.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	{"text": "How's this for a\nband name: _______\nand the _______s!", "numResponses": 2},
 	{"text": "Imma let you finish,\nbut _______ had the\nbest _______ of all\ntime.", "numResponses": 2},
 	{"text": "Make America\n_______ Again!"}
+	{"text": "Why did Grumpy\nthreaten a\nsmall child?"}
```
Comment (GrumpyCrouton) [inline decks/altspace.json:11]: I just wanna be famous :'(

**Comment by stevenvergenz** [comment]
That's horrifying, I love it!

Some syntax and spacing errors though. I'll straighten those out, then merge.

**Comment by GrumpyCrouton** [comment]
Ah, I've just noticed the missing commas. I'll get better at the spacing, I need to get a feel for how big the cards are even more than I do.

So I have a question, in AltSpace, any changes made here will automatically be put into the game in AltSpace, or do they need to do updates?

I'll be doing a bigger pull request later with more cards.

**Comment by stevenvergenz** [comment]
If you have any dev experience and can set up a local server, I have a card preview app for debugging the spacing. Start the server, and go to http://localhost:7878/static/testCards.html.

The live game is run from our servers, which are set up to automatically deploy changes to the master branch. If you want to run a private game, point your enclosure to http://hah.nodriftreality.com/play.

**Comment by GrumpyCrouton** [comment]
I have a bit of dev experience, I am the same Grumpy that spoke to you last night in the tavern. That's really neat!

Once I get home from work I will probably do that.

**Comment by stevenvergenz** [comment]
Ah that's right! Please forgive my terrible memory :p

**Comment by GrumpyCrouton** [comment]
It's all good :P

**Pull request merged**